src/Robo/Plugin/Commands/EnvSwitchCommand.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Fire\Robo\Plugin\Commands;
+
+use Fire\Robo\Plugin\Commands\FireCommandBase;
+use Robo\Symfony\ConsoleIO;
+use Robo\Robo;
+
+/**
+ * Provides a command to Switch between the local_environment.
+ */
+class EnvSwitchCommand extends FireCommandBase {
+
+  /**
+   * Add a new command.
+   *
+   * Usage Example: fire switch
+   *
+   * @command env:switch
+   * @aliases env-switch, switch, sw
+   * @usage fire switch
+   */
+  public function switch(ConsoleIO $io) {
+    $tasks = $this->collectionBuilder($io);
+    $env = Robo::config()->get('local_environment');
+    $new_env = ($env === 'lando') ? 'ddev' : 'lando';
+    $filename = 'fire.local.yml';
+    $new_content = "local_environment: $new_env";
+
+    if (file_exists($filename)) {
+      $content = file_get_contents($filename);
```

Comment:
Instead of doing this please use the Symfony's Yaml component to load and write yaml files , example:
https://github.com/fourkitchens/fire/blob/2.x/src/Robo/Plugin/Commands/VrtLocalEnvConfigureCommand.php#L26

Reply:
Hi @rigoucr done!